app/models/notification_type.rb

```diff
@@ -27,9 +29,17 @@ def subscriber_ids(subject, initiator)
       end.try(:user_ids)
     when AUDIENCE_SUPERADMIN
       User.superadmins.pluck(:id)
+    when AUDIENCE_NONE
+      []
     end
   end
 
+  # this disables notifications, but allows the notification to still exist
+  # this notification template can be used for emails
+  def none?
```

Comment:
I don't really like the name. At first glance it feels like `Enumerable#none?` which is completely unrelated.

Maybe `has_audience?` or even something like `enabled?` (or the negative of either of those)?